README.md

```diff
@@ -23,11 +23,11 @@
 ### Using the CommonLibSF plugin templates
 
 - [CLibSFPluginTemplate](https://github.com/Starfield-Reverse-Engineering/CLibSFPluginTemplate)
-  - Uses **CMake**. Consumes CommonLibSF as a [vcpkg port package](https://github.com/Starfield-Reverse-Engineering/Starfield-RE-vcpkg) or git submodule.
+  - Uses **CMake**. Consumes CommonLibSF as a [vcpkg port package](https://github.com/Starfield-Reverse-Engineering/Starfield-RE-vcpkg) or a git submodule.
 - [SF_PluginTemplate](https://github.com/gottyduke/SF_PluginTemplate)
-  - Uses **CMake**. Consumes CommonLibSF as a git submodule or local fork.
+  - Uses **CMake**. Consumes CommonLibSF as a git submodule or a local fork.
 - [commonlibsf-template-xmake](https://github.com/Starfield-Reverse-Engineering/commonlibsf-template-xmake)
-  - Uses **XMake**. Consumes CommonLibSF as an [xrepo package](https://github.com/Starfield-Reverse-Engineering/commonlibsf-xrepo).
+  - Uses **XMake**. Consumes CommonLibSF as a [xrepo package](https://github.com/Starfield-Reverse-Engineering/commonlibsf-xrepo), git submodule, or a local fork.
```

Comment:
"an" is grammatically correct; "a" doesn't work before "xrepo" because the latter starts with a vowel sound